syntax = "proto3";

package order_queue;

message OrderQueueRequest {
    bool is_valid = 1;
    string orderID = 2;
    string[] queue = [];
}

message OrderQueueResponse {
    bool is_in_queue = 1;
    string message = 2;
    string[] queue = [];
}

service OrderQueueService {
   rpc EnqueueOrder(OrderQueueRequest) returns (OrderQueueResponse) {}
}